### ISSUE TITLE
hypersim_anchor_swins results

### ISSUE THREAD

**Issue by Lawrence2025** [lyclyc52/NeRF_RPN#14]
Dear authors,

Thanks for your excellent work!

I tried to evaluate the accuracy of your pre-trained model on hypersim, the script is like:

`DATA_ROOT=/xxx/hypersim_rpn_data

python3 -u run_rpn.py \
--mode "eval" \
--dataset_name hypersim \
--resolution 160 \
--backbone_type swin_s \
--features_path ${DATA_ROOT}/features \
--boxes_path ${DATA_ROOT}/obb \
--dataset_split ${DATA_ROOT}/hypersim_split.npz \
--save_path ./results/hypersim_anchor_swins \
--checkpoint ./results/hypersim_anchor_swins/hypersim_anchor_swinS.pt \
--rpn_nms_thresh 0.3 \
--normalize_density \
--rotated_bbox \
--batch_size 2 \
--gpus 0 `

and the results are like:

Top 300 proposals:
Recall@50: Recall: 0.0317, Num pos: 315
Recall@25: Recall: 0.2286, Num pos: 315
AR: 0.0523

Top 1000 proposals:
Recall@50: Recall: 0.0381, Num pos: 315
Recall@25: Recall: 0.3143, Num pos: 315
AR: 0.0677

Top 2500 proposals:
Recall@50: Recall: 0.0413, Num pos: 315
Recall@25: Recall: 0.3524, Num pos: 315
AR: 0.0745
AP@50: AP: 0.0001
AP@25: AP: 0.0274

It seems that the results are not the same as reported in the paper. Is there anything wrong with my test process?

Best

**Comment by zymk9** [comment]
Hi. The problem is likely due to the `--normalize_density` flag. As mentioned [here](https://github.com/lyclyc52/NeRF_RPN#use-pretrained-model-weights) and [here](https://github.com/lyclyc52/NeRF_RPN/blob/main/data/README.md#nerf-rpn-data-organization), the density extracted from Hypersim has already been normalized (i.e., converted to alpha) whereas for 3D-FRONT and ScanNet this is not the case. 

So if you are using the pre-trained checkpoint we provided, the `--normalize_density` flag should be removed. If you are training from scratch with the flag enabled the performance should not be that bad, but can still be different from what is reported in the paper.

Sorry for this huge confusion. Hope that helps.

**Comment by Lawrence2025** [comment]
Thanks a lot!
